models/yang/annotations/openconfig-interfaces-annot.yang

```diff
@@ -139,11 +139,13 @@ module openconfig-interfaces-annot {
 
     deviation /oc-intf:interfaces/oc-intf:interface/oc-intf:subinterfaces/oc-intf:subinterface/oc-ip:ipv4/oc-ip:addresses {
         deviate add {
+            sonic-ext:table-name "NONE";
```

Comment:
@kwangsuk Can we handle this in the tranformer itself? I dont see a need to call any other xfmr functions when subtree is defined.